[2025-05-07T01:30:20.802+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-07T01:30:20.835+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [queued]>
[2025-05-07T01:30:20.851+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [queued]>
[2025-05-07T01:30:20.866+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-07T01:30:20.890+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): generate_dbt_docs> on 2025-05-07 01:02:20.058495+00:00
[2025-05-07T01:30:20.902+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=12727) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-07T01:30:20.905+0000] {standard_task_runner.py:72} INFO - Started process 12742 to run task
[2025-05-07T01:30:20.909+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'generate_dbt_docs', 'manual__2025-05-07T01:02:20.058495+00:00', '--job-id', '747', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp4zt0lh6v']
[2025-05-07T01:30:20.912+0000] {standard_task_runner.py:105} INFO - Job 747: Subtask generate_dbt_docs
[2025-05-07T01:30:20.954+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.generate_dbt_docs manual__2025-05-07T01:02:20.058495+00:00 [running]> on host 8280c28b7f0f
[2025-05-07T01:30:21.022+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='generate_dbt_docs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T01:02:20.058495+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T01:02:20.058495+00:00'
[2025-05-07T01:30:21.024+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-07T01:30:21.025+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-07T01:30:21.025+0000] {logging_mixin.py:190} INFO - Current task name:generate_dbt_docs state:running start_date:2025-05-07 01:30:20.836104+00:00
[2025-05-07T01:30:21.028+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-07T01:30:21.028+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-07T01:30:21.029+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-07T01:30:21.030+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt docs generate']
[2025-05-07T01:30:21.047+0000] {subprocess.py:99} INFO - Output:
[2025-05-07T01:30:23.156+0000] {subprocess.py:106} INFO - [0m01:30:23  Running with dbt=1.9.4
[2025-05-07T01:30:23.469+0000] {subprocess.py:106} INFO - [0m01:30:23  Registered adapter: postgres=1.9.0
[2025-05-07T01:30:23.996+0000] {subprocess.py:106} INFO - [0m01:30:23  Found 5 models, 12 data tests, 433 macros
[2025-05-07T01:30:24.000+0000] {subprocess.py:106} INFO - [0m01:30:24
[2025-05-07T01:30:24.002+0000] {subprocess.py:106} INFO - [0m01:30:24  Concurrency: 1 threads (target='dev')
[2025-05-07T01:30:24.002+0000] {subprocess.py:106} INFO - [0m01:30:24
[2025-05-07T01:30:24.686+0000] {subprocess.py:106} INFO - [0m01:30:24  Building catalog
[2025-05-07T01:30:24.817+0000] {subprocess.py:106} INFO - [0m01:30:24  Catalog written to /opt/***/dbt_project/bank_reviews_dbt/target/catalog.json
[2025-05-07T01:30:27.421+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-07T01:30:27.455+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-07T01:30:27.456+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=generate_dbt_docs, run_id=manual__2025-05-07T01:02:20.058495+00:00, execution_date=20250507T010220, start_date=20250507T013020, end_date=20250507T013027
[2025-05-07T01:30:27.479+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-07T01:30:27.480+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-07T01:30:27.481+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-07 01:02:20.137662+00:00
[2025-05-07T01:30:27.481+0000] {logging_mixin.py:190} INFO - Task hostname:8280c28b7f0f operator:BashOperator
[2025-05-07T01:30:27.504+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-07T01:30:27.521+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-07T01:30:27.523+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
